vit-servicing-station-lib/migrations/2020-05-22-112032_initial/up.sql

```diff
@@ -34,7 +34,8 @@ create table proposals
     chain_proposal_id BLOB NOT NULL,
     chain_proposal_index BIGINT NOT NULL,
     chain_vote_options VARCHAR NOT NULL,
-    chain_voteplan_id VARCHAR NOT NULL
+    chain_voteplan_id VARCHAR NOT NULL,
+    challenge_id INTEGER NOT NULL
```

Comment:
Same applies here: it should be `alter table` in a new migration subdirectory, starting from templates created by the diesel CLI tool.